src/bid/mod.rs

```diff
@@ -5,5 +5,6 @@
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
 pub use bid::Bid;
+pub use bid::BID_SIZE;
```

Comment:
I think the current approach is having the constant function `serialized_size` directly on the type.

We should refactor probably the previous code, but for new code we should follow this direction.

(Also, not related to this PR, we should port the path modules to Rust 2018, is there an issue for that?)

Reply:
No, but I'll add it once we close this